SPECS/python-psycopg2/python-psycopg2.spec

```diff
@@ -43,6 +43,15 @@ Requires:       postgresql >= 10.5
 %description -n python3-psycopg2
 Python 3 version.
 
+%package doc
+Summary:	Documentation for psycopg python PostgreSQL database adapter
+%{?with_python2:Provides: python2-%{srcname}-doc = %{version}-%{release}}
```

Comment:
We don't use the `with_X` macros too extensively. Please just keep the `Provides` for Python 2 and remove the macro logic.

Reply:
Sure